### ISSUE TITLE
Language list 2016-06-29_138 is missing -- deleted?

### ISSUE THREAD

**Issue by PaulHeggarty** [lingdb/CoBL#221]
http://www.cobl.info/languagelist/2016-06-29_138/
gives a page not found error, and the _138 list is not listed under http://www.cobl.info/languagelist/
(only the 138_alpha list is).
This is our default list, so we need it back!

**Comment by runjak** [comment]
I've just had a conversation per mail about this with @cormacanderson - I should probably have you copied in.
The thing is that Cormac renamed the language list but the current system knows defaults only by their name.
The quickfix for this will be to simply adjust the default name to be the same as the language list again.
The real improvement will be some selection for defaults on the website so that default lists can be changed without the need for code adjustments.

**Comment by runjak** [comment]
I'll deploy the simple quickfix for this now so that users don't get too irritated by the current situation and afterwards tackle the general problem.

**Comment by PaulHeggarty** [comment]
This is not the the first time we've had this problem.  We really do need to have a failsafe system where we can change the name of a list without it disrupting things like this.  I think language lists need to have separate fields for `System Name` and `Display Name`, so that we can change the latter but the system name remains stable.

**Comment by cormacanderson** [comment]
We really don't want to use the "all" list for anything, if we are to maintain credibility in the project. We could default to the "Reference" list maybe - that wouldn't be so bad. I suppose we could just rename the default list "Current" and abandon the practice of trying to include the number of languages in the language name.

**Comment by PaulHeggarty** [comment]
Basically, the only people who should be ablthe to see **any** list other than the current default are the admin people, surely?  I’m actually wondering why we need to bother with lists much nowadays, except for exports.  They are certainly causing us far more trouble than they are worth.  We need to sit down and have a proper chat on a long-term solution for this at some point, but not until August will we have time, there are other priorities first.  But I do think that for all non-admins, switching off all lists except the default should be done **now**.

**Comment by PaulHeggarty** [comment]
Yes, we should just have a list called `Current`, and make changes to it as we want, without renaming it constantly.  There is no point to a list other than keeping it for exports, so we don’t really need any new lists since our last export.  When we do an export, we just clone the `Current` list and save the clone with the date and number of languages in that export.  The `Current` list goes on and keeps getting changed, the dated lists are just snapshots of the current list at various stages of development.

**Comment by PaulHeggarty** [comment]
I’m also unhappy at having a `Current` list that includes totally unrevised legacy languages without that being clear to people. I think that now that we can rename language Display Names, we could even prefix the bad ones with something like NR: for 'Not Revised'. So `NR: Assamese`, for example. We could even do this immediately, and it would be a relief.

**Comment by cormacanderson** [comment]
I see the sense in that, although remember there is still the popup security guard, which means that this data isn't freely available to the whole world yet. Another option would be to make the progress bars visible to everyone, and to display them prominently also at the top of the `1Mng, All Lgs` page.
